_pages/model-deployment/allennlp.md

```diff
@@ -17,7 +17,7 @@ Welcome to deploying your <a href="https://allennlp.org/">AllenNLP</a> model on
 
 This guide is designed as an introduction to deploying a AllenNLP model and publishing an algorithm even if you’ve never used Algorithmia before.
 
-Note: this guide uses the web UI to create and deploy your Algorithm. If you prefer a code-only approach to deployment, review [Algorithm Management API]({{site.baseurl}}/algorithm-development/algorithm-management-api) after reading this guide.
+Note: this guide uses the web UI to create and deploy your Algorithm. If you prefer a code-only approach to deployment, review [algorithm management]({{site.baseurl}}/algorithm-development/algorithm-management) after reading this guide.
```

Comment:
:bulb: Might update this to "our algorithm management guide".

Reply:
Or perhaps capitalize it, just to keep it consistent with the prior link.

Reply:
Updated